openedx/core/djangoapps/api_admin/forms.py

```diff
@@ -42,3 +28,16 @@ class Meta(object):
         widgets = {
             'company_address': forms.Textarea()
         }
+
+    def __init__(self, *args, **kwargs):
+        super(ApiAccessRequestForm, self).__init__(*args, **kwargs)
+        self.fields['terms_of_service'].label = mark_safe(
```

Comment:
Note: I spent far too long trying to get this to work in the "usual" way. Since there's HTML in the label, we need `mark_safe`. Since there's a URL in the link, we need to use `reverse`. But when specifying that in the class definition, we need to use `reverse_lazy` since the urlconf isn't loaded yet. This has weird interactions with `mark_safe`, so it seems like it's cleanest to explicitly delay evaluating the URL until form init time.